### PR TITLE
Create stub task rest routes

### PR DESCRIPTION
**Pull request by merekorlowski**

### CODE REVIEW

File: src/client/src/components/Content/Schedule/Calendar/Day/styles.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 .day {
-    display: table-cell;
+  display: table-cell;
 }
 
 .day li {
-    display: block;
```
Comment (karim2506) [inline src/client/src/components/Content/Schedule/Calendar/Day/styles.css]: I can see that you fixed the indentation for the js files (you made it tab size 4 if I'm not mistaken).

The indentation for the css files will be 2 spaces or...?

Comment (merekorlowski) [reply src/client/src/components/Content/Schedule/Calendar/Day/styles.css]: It's actually tab 2 everywhere, in the pull request it doesn't show but if you look in the code of GitHub it looks good.

File: src/client/src/models/Task.js [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
-        this.inProgress = false;
-        this.finished = false;
-        this.timeLogs = [];
-    }
+	constructor(id, title, description, colour) {
+		this.id = id;
+		this.title = title;
+		this.description = description;
+		this.colour = colour;
+		this.status = status;
```
Comment (karim2506) [inline src/client/src/models/Task.js]: Where does the status parameter come from?

Comment (merekorlowski) [reply src/client/src/models/Task.js]: True, I have not yet updated the task model from when we originally made it. Will do this.

**Pull request merged**